Sources/StreamChatUI/CommonViews/ChatSuggestions/ChatSuggestionsCollectionView.swift

```diff
@@ -6,10 +6,10 @@ import StreamChat
 import UIKit
 
 /// The collection view of the suggestions view controller.
-public typealias ChatMessageComposerSuggestionsCollectionView = _ChatMessageComposerSuggestionsCollectionView<NoExtraData>
+public typealias ChatSuggestionsCollectionView = _ChatSuggestionsCollectionView<NoExtraData>
 
 /// The collection view of the suggestions view controller.
-open class _ChatMessageComposerSuggestionsCollectionView<ExtraData: ExtraDataTypes>: UICollectionView,
+open class _ChatSuggestionsCollectionView<ExtraData: ExtraDataTypes>: UICollectionView,
```

Comment:
I think this name is a bit generic, and hard to understand what are these suggestions about. WDYT of `ChatInputSuggestions...` prefix ?

Reply:
I don't think we need/should have this type tbh. It's just a collection view with some layout settings. The setting should happen in the VC, aynways.

Reply:
Yes, that is true, this one not necessary at all probably. My comment is only for the renaming of these components prefix, to be `InputSuggestions` instead of only `Suggestions`.

Reply:
valid feedback, i think it's better suited for the UIConfig follow-up PR